gateway-provider-security-pac4j/src/main/java/org/apache/knox/gateway/pac4j/filter/Pac4jDispatcherFilter.java

```diff
@@ -190,8 +190,18 @@ private void addDefaultConfig(String clientNameParameter, Map<String, String> pr
       properties.put(PropertiesConfigFactory.SAML_KEYSTORE_PATH,
           keystoreService.getKeystorePath());
 
-      properties.put(PropertiesConfigFactory.SAML_KEYSTORE_PASSWORD,
-          new String(masterService.getMasterSecret()));
+      // check for provisioned alias for keystore password
+      char[] giksp = null;
```

Comment:
It is possible, but I was trying to keep with the style of the file (and method impl) where `gip` (Gateway Identity Password) was already used.  In an effort to not add more lines to the patch, I left `giksp` and `gip` rather than changing both to `gatewayIdentityKeystorePassword` and `gatewayIdentityPassword` or `identityKeystorePassword` and `identityPassword`, or whatnot.